src/store/calendarObjectInstance.js

```diff
@@ -1424,7 +1438,12 @@ const actions = {
 		const calendarObject = await dispatch('createNewEvent', { start, end, isAllDay, timezoneId })
 		const startDate = new Date(start * 1000)
 		const eventComponent = getObjectAtRecurrenceId(calendarObject, startDate)
-		const calendarObjectInstance = mapEventComponentToEventObject(eventComponent)
+		const eventProps = objectFromEntries(Object.entries({ title, description }).filter(([k, v]) => v != null))
```

Comment:
```suggestion
		if (title) {
			eventComponent.title = title
		}
		if (description) {
			eventComponent.description = description
		}
		const calendarObjectInstance = mapEventComponentToEventObject(eventComponent)
```

If you add the properties to `eventComponent` like shown above, `mapEventComponentToEventObject` will automatically include them in `calendarObjectInstance`.

Reply:
Fixed. thanks. I just used `title != null` comparison instead if that's ok as it's probably more clear.

It doesn't matter in js for this case as the query string is string: https://developer.mozilla.org/en-US/docs/Glossary/Falsy

But e.g. php (also used in this projects) considers `"0"` to be falsy: https://www.php.net/manual/en/language.types.boolean.php and I find it hard to remember each languages boolean conversions.